scripts/pred/call_api.py

```diff
@@ -84,7 +85,7 @@ def __call__(self, parser, namespace, values, option_string=None):
 parser.add_argument("--random_seed", type=int, default=0)
 parser.add_argument("--stop_words", type=str, default='')
 parser.add_argument("--sliding_window_size", type=int)
-parser.add_argument("--threads", type=int, default=4)
```

Comment:
Can you help restore our default setting since people can change their threads based their needs?